gmp.spec

```diff
@@ -1,2 +1,2 @@
-### RPM external gmp 6.1.2
-Source: ftp://mirrors.kernel.org/gnu/%n/%n-%realversion.tar.xz
+### RPM external gmp 6.2.0
```

Comment:
do we have any thing using this? I mean any package depends on gmp or all of our packages use gmp-static?

Reply:
only in this file:
cmssw-queue-override.file
```
%if "%(case %realversion in (*_ICC_X*) echo true ;; (*) echo false ;; esac)" == "true"                                                                                    
%define gitcommit       %(echo %realversion | sed -e 's|_ICC_X|_X|')                                                                                                      
%define scram_compiler  icc                                                                                                                                               
%define extra_tools     mpfr gmp icc-cxxcompiler icc-f77compiler icc-ccompiler                                                                                            
%endif 
```

also there is elfutils.spec not use by anything.

Reply:
this `gmp` tool is defined in gmp-static ( https://github.com/cms-sw/cmsdist/blob/IB/CMSSW_11_1_X/master/gmp-static-toolfile.spec ) , so I think no one is using gmp.spec. So better to delete it.